src/App.style.ts

```diff
@@ -112,4 +113,73 @@ export const StyledFormGroup = styled(FormGroup)`
     gap: 20px;
     color: white;
 `
+export const StyledDetail = styled.div`
+    position: relative;
+    top: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    padding: 10px;
+    background: rgba(219, 212, 60, 0.76);
```

Comment:
above you're using hex for colors, and here rgb, please unify styles to be consistent and lets use one of them.

Reply:
unified